subprojects/platform-play/src/testFixtures/groovy/org/gradle/play/integtest/fixtures/DistributionTestExecHandleBuilder.groovy

```diff
@@ -40,6 +40,8 @@ class DistributionTestExecHandleBuilder {
         this.setExecutable("${baseDirName}/playBinary/bin/playBinary${extension}")
         this.environment("PLAY_BINARY_OPTS": "-Dhttp.port=${port}")
         this.environment("JAVA_HOME", System.getProperty("java.home"))
+        // https://github.com/gradle/dotcom/issues/6071
+        this.environment("JAVA_OPTS", '')
```

Comment:
From the linked issue, it looks like this was fixed in `1.0.1`. If it is still need, can you explain why in the comment?

Reply:
That's test-distribution-agent 1.0.1, not test-distribution-plugin 1.0.1.